src/main/java/core/basesyntax/Truck.java

```diff
@@ -0,0 +1,11 @@
+package core.basesyntax;
+
+public class Truck extends Machine {
+    public void doWork() {
+        System.out.println("Truck is now working.");
+    }
+
+    public void stopWork() {
+        System.out.println("Truck has stopped working.");

```

Comment:
Similarly, the stopWork message should clearly indicate that it is a 'Truck' that has stopped working. For instance: 'Truck has finished its work.'